packages/openactive-integration-tests/test/features/notifications/customer-notice-notifications/implemented/customer-notice-notification-test.js

```diff
@@ -0,0 +1,101 @@
+  const requestHelper = new RequestHelper(logger);
+
+  // ## Initiate Flow Stages
+  const defaultFlowStageParams = FlowStageUtils.createDefaultFlowStageParams({ requestHelper, logger });
+  const fetchOpportunities = new FetchOpportunitiesFlowStage({
+    ...defaultFlowStageParams,
+    orderItemCriteriaList,
+  });
+  const c1 = new C1FlowStage({
+    ...defaultFlowStageParams,
+    prerequisite: fetchOpportunities,
+    getInput: () => ({
+      orderItems: fetchOpportunities.getOutput().orderItems,
+    }),
+  });
+  const c2 = new C2FlowStage({
+    ...defaultFlowStageParams,
+    prerequisite: c1,
+    getInput: () => ({
+      orderItems: fetchOpportunities.getOutput().orderItems,
+    }),
+  });
+  const b = new BFlowStage({
+    ...defaultFlowStageParams,
+    prerequisite: c2,
+    getInput: () => ({
+      orderItems: fetchOpportunities.getOutput().orderItems,
+      totalPaymentDue: c2.getOutput().totalPaymentDue,
+    }),
+  });
+  const [simulateCustomerNoticeUpdate, orderFeedUpdate] = OrderFeedUpdateFlowStageUtils.wrap({
+    wrappedStageFn: prerequisite => (new TestInterfaceActionFlowStage({
+      ...defaultFlowStageParams,
+      testName: 'Simulate Customer Notice Update (Test Interface Action)',
+      prerequisite,
+      createActionFn: () => ({
+        type: 'test:CustomerNoticeSimulateAction',
+        objectType: 'Order',
+        objectId: b.getOutput().orderId,
+      }),
+    })),
+    orderFeedUpdateParams: {
+      ...defaultFlowStageParams,
+      prerequisite: b,
+      testName: 'Orders Feed (after Simulate Customer Notice Update)',
+    },
+  });
+
+  // ## Set up tests
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(fetchOpportunities);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(c1);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(c2);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(b);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(simulateCustomerNoticeUpdate);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(orderFeedUpdate, () => {
+    it('should have customer notices with non empty string values', () => {
+      // new = after the CustomerNoticeSimulateAction was invoked
+      const newOrderItems = orderFeedUpdate.getOutput().httpResponse.body.data.orderedItem;
+      // As we'll be setting out expectations in an iteration, this test would
+      // give a false positive if there were no items in `orderedItem`, so we
+      // explicitly test that the OrderItems are present.
+      expect(newOrderItems).to.be.an('array');
+      expect(newOrderItems).to.have.lengthOf.above(0);
+      expect(newOrderItems).to.have.lengthOf(orderItemCriteriaList.length);
+
+      for (const newOrderItem of newOrderItems) {
+        const newCustomerNotice = newOrderItem.customerNotice;
+
+        // Checking that values of customer notices are altered.
+        expect(newCustomerNotice).to.be.a('string');
+        /* eslint-disable no-unused-expressions */
```

Comment:
@lukehesluke @nickevansuk 
Can one of you confirm is ok to disable this lint error when checking that string is not empty?
Or should it be checked with some other command?